test/actions/katello/repository_test.rb

```diff
@@ -444,7 +444,8 @@ class CapsuleGenerateAndSyncTest < TestBase
 
     it 'plans' do
       plan_action(action, repository)
-      assert_action_planed_with(action, ::Actions::Katello::CapsuleContent::Sync, capsule_content, :repository => repository)
+      assert_action_planned_with(action, ::Actions::BulkAction, ::Actions::Katello::CapsuleContent::Sync,
+                                 [capsule_content.capsule], :repository_id => repository.id)
     end
```

Comment:
How hard would it be to test two proxies syncing using bulk actions? That is some new functionality added by this PR and it would be nice to have the test coverage.